packages/docs/src/helpers/SortObjectByPropNamesHelper.ts

```diff
@@ -0,0 +1,9 @@
+export const sortObjectByPropNames = (obj: any) => {
```

Comment:
No `any`s please. Better use `Record<string, unknown>`, or even something like this but in conjunction with generics.